packages/create-svelte/templates/default/src/routes/sverdle/+page.svelte

```diff
@@ -109,10 +109,12 @@
 	<a class="how-to-play" href="/sverdle/how-to-play">How to play</a>
 
 	<div class="grid" class:playing={!won} class:bad-guess={form?.badGuess}>
+		<!-- eslint-disable-next-line no-unused-vars -->
```

Comment:
I don't think this is the best way but at the same time, I think we shouldn't change the template ESLint config for this. Therefore I choose to add ESLint disable for this line.

Reply:
Is there any way to not show this warning in eslint when there's an underscore? Right now you can't omit the variable, so you can't fix this warning.

Reply:
https://eslint.org/docs/latest/rules/no-unused-vars#argsignorepattern

Reply:
Yeah, there is a configuration, but I don't think we should add this config on template files.

```js
rules: {
  'no-unused-vars': [
    'error',
    {
      varsIgnorePattern: '^_'
    }
  ]
}
```

So instead of this, I updated the code itself.
https://github.com/sveltejs/kit/pull/9749/commits/1e4c26cfaf6a9f475ff39f5feeccc6c70687d392